webapp/channels/src/store/index.ts

```diff
@@ -14,15 +16,34 @@ import {clearUserCookie} from 'actions/views/cookie';
 import appReducers from 'reducers';
 import {getBasePath} from 'selectors/general';
 
+import type {GlobalState} from 'types/store';
+
 function getAppReducers() {
     return require('../reducers'); // eslint-disable-line global-require
 }
 
+declare global {
```

Comment:
Having these in-line feels wrong to me - it makes it hard to be discoverable. I think this should be in a `.d.ts` file.

Reply:
I kinda like it to have it here. I don't know if we want to get rid of zen-observable, and zen-observable is only used in this file, so probably if we move that to other place, if we remove zen-observable, we end up leaving that there. I agree that it is an smell, but I think in this case is the right option.